src/qtgui/iq_tool.h

```diff
@@ -82,9 +82,16 @@ private slots:
     void timeoutFunction(void);
 
 private:
+    enum IqToolState
+    {
+        idle = 0,
+        playing,
+        recording
```

Comment:
NIT: The other enums in this project are UPPERCASE and most of them have `enum` name preffix, pls can you do the adjustment?

eg.  https://github.com/gqrx-sdr/gqrx/blob/master/src/qtgui/bookmarkstablemodel.h#L37  

(personally I would use `STATE_*` here, not full name) 

Otherwise LGTM.

Reply:
No problem. Updated.